.github/workflows/_swift.yml

```diff
@@ -117,14 +117,12 @@ jobs:
           # Needed because `productbuild` doesn't support picking this up automatically like Xcode does
           INSTALLER_CODE_SIGN_IDENTITY: "3rd Party Mac Developer Installer: Firezone, Inc. (47R2M6779T)"
           REQUESTED_XCODE_VERSION: ${{ matrix.xcode }}
-          # TODO: Change this to "prod" when app is released
-          XCCONFIG_ENV: staging
         run: |
           # Set Xcode version to use if provided
           [[ ! -z "$REQUESTED_XCODE_VERSION" ]] && sudo xcode-select -s /Applications/Xcode_$REQUESTED_XCODE_VERSION.app
 
           # Copy xcconfig
-          cp Firezone/xcconfig/${{ env.XCCONFIG_ENV }}.xcconfig Firezone/xcconfig/config.xcconfig
+          cp Firezone/xcconfig/release.xcconfig Firezone/xcconfig/config.xcconfig
```

Comment:
@roop With the runtime settings removed from `xcconfig` we can simplify the xcconfigs even further to just `debug` and `release`

Reply:
Agreed.